packages/components/src/components/Radio/ControlledRadioGroup/ControlledRadioGroup.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Control, Controller } from "react-hook-form";
+import { RadioGroup, RadioGroupProps } from "../RadioGroup";
+
+export interface ControlledRadioGroupProps extends RadioGroupProps {
```

Comment:
We probably don't want people passing in `onValueChange` or `value` props since that would then override what's coming from the Controller.